benchmarks/nemo/nemolite2d/manual_versions/psykal_kokkos/Makefile

```diff
@@ -41,17 +74,19 @@ include $(KOKKOS_PATH)/Makefile.kokkos
 # called from C++ code
 %.exe: $(MODULE_LIST) $(KOKKOS_LINK_DEPENDS) 
 	$(F90) -o $@ $(MODULE_LIST) ${COMMON_LIB} ${INF_LIB} $(TIMER_LIB) \
-		$(LDFLAGS) ${OMPFLAGS} $(KOKKOS_LINK_DEPENDS) $(KOKKOS_LDFLAGS) $(KOKKOS_LIBS) -lstdc++
+		$(LDFLAGS) ${OMPFLAGS} $(KOKKOS_LINK_DEPENDS) $(KOKKOS_LIBS) \
+		-L/opt/nvidia/cuda-10/lib64 -L. -lstdc++
```

Comment:
Sorry to be a pain but we should parametrise the location of the CUDA library (maybe there's already an environment variable we can query?)

Reply:
I agree, I meant to change this before pushing the PR but I forgot. I found a way to deduce the CUDA library path which seems to work well.